config/locales/en.yml

```diff
@@ -456,9 +456,9 @@ en:
             will_need_chromebooks:
               blank: Tell us whether the school will need Chromebooks
```

Comment:
Can we change "school" to school.institution_type?

Reply:
I've got this working in 7bc68bbe, and refactored `spec/form_objects/chromebook_information_form_spec.rb` to test the new behaviour - can you take another look?